src/index.ts

```diff
@@ -0,0 +1,6 @@
+import { createDefaultHttpTrigger, ConfigurationParams } from 'perimeterx-azure-js-sdk';
+import config from './enforcerConfig.json';
+
+const httpTrigger = createDefaultHttpTrigger(config as ConfigurationParams, process.env.FRONT_DOOR_SECRET_KEY);
```

Comment:
Maybe add a separate file for custom functions too?

Reply:
It is currently aligned with the other enforcers' example projects, so we can add it next time (maybe as part of the custom code task).